app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -388,7 +388,7 @@ def import_uploads
     param :id, :identifier, :required => true
     def gpg_key_content
       if @repository.gpg_key && @repository.gpg_key.content.present?
-        render(:text => @repository.gpg_key.content, :layout => false)
+        render(:plain => "#{@repository.gpg_key.content}", :layout => false)
```

Comment:
Thanks @jlsherrill 
Changing the code and refreshing the browser was actually giving a 304 status and not actually re-rendering the new response leading to my confusion about root cause.
Verified that interpolation is unnecessary here. Pushed the requested change. :+1: